.env.jenkins

```diff
@@ -6,7 +6,7 @@ ETHEREUM_DEV_CLUSTER=1
 EXTENSIONS=1
 FLEET=eth.beta
 GROUP_CHATS_ENABLED=1
-HARDWALLET_ENABLED=0
+HARDWALLET_ENABLED=1
```

Comment:
do we want to keep it enabled?

Reply:
@mandrigin this is for PR testing i believe, not for nightly builds

Reply:
Okay, just to mention that if that will be merged, every PRbuild will have keycard on :) I think we really should move these from flags to checkboxes in the dev mode.

Reply:
@mandrigin after PR is tested, i disable that flag back before merge. But it might be good idea with checkbox in dev mode, i'll check that.